doc/fcn_2D_segm.txt

```diff
@@ -0,0 +1,210 @@
+.. _fcn_2D_segm:
+
+Fully Convolutional Networks (FCN) for 2D segmentation
+******************************************************
+
+Summary
++++++++
+
+Segmentation task is different from classification task because it requires predicting
+a class for each pixel of the input image, instead of only 1 class for the whole input.
+Classification needs to understand *what* is in the input (namely, the context). However,
+in order to predict what is in the input for each pixel, segmentation needs to recover
+not only *what* is in the input, but also *where*.
+
+.. figure:: images/cat_segmentation.png
+    :align: center
+    :scale: 35%
+
+    **Figure 1** : Segmentation network (from FCN paper)
+
+TODO : reference de l'image
+
+**Fully Convolutional Networks** (FCNs) owe their name to their architecture, which is
+built only from locally connected layers, such as convolution, pooling and upsampling.
+Note that no dense layer is used in this kind of architecture. This reduces the number
+of parameters and computation time. To obtain a segmentation map (output), segmentation
```

Comment:
Also, the network can work regardless of the original image size, since all connections are local, there does not need to be any fixed number of units at any stage.